.github/workflows/update-test-times.yml

```
@@ -23,14 +23,22 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: Set up python 3.10
+        uses: actions/setup-python@v2
```

Why using `v2` when `v5` is available?
```suggestion
        uses: actions/setup-python@v5
```